### DISCUSSION TITLE
Futures support

### DISCUSSION THREAD

**Discussion by AiyuJaox** [DogsTailFarmer/martin-binance#80] [General]
Hello, it's really a good strategy. Do you have the plan to add the support for the futures market?

**Comment by DogsTailFarmer** [comment]
Hi!

I don't plan to do this.

Regards Jerry.

**Comment by adamantmm** [comment]
Fair call on skipping futures for now — the strategy assumptions here are spot-native.

Futures would force a different risk model: funding rates, mark vs last, hedge vs one-way position mode, and liquidation distance all sit outside the current grid / martin sizing. Mapping `AMOUNT_SECOND` style spot inventory onto a leveraged book without those knobs is how a lot of ports blow up.

If someone forks for USDT-M later, I would keep the grid math on notional and treat leverage/margin as a separate adapter, not bake it into the step-size formulas.
